tools/build.sh

```diff
@@ -64,60 +64,60 @@ echo -en "\noption java_multiple_files = true;" >> $MonitoredResourceProto
 echo -en "\noption java_outer_classname = \"MonitoredResourceProto\";" >> $MonitoredResourceProto
 echo -en "\noption java_package = \"com.google.api\";" >> $MonitoredResourceProto
 
-_generate() {
-    proto_src=$1
-    echo
-    echo "########################################################"
-    echo "# Generating from $proto_src"
-
-    # Comment out preset Java options
-    sed -i $MACOS 's/option java/\/\/option java/' $proto_src
-    # Set Java option: multiple files
-    echo -en "\noption java_multiple_files = true;" >> $proto_src
-    # Set Java option: package name
-    awk '/^package/,/;/' $proto_src > PROTO_PACKAGE; # Ex. package google.events.cloud.pubsub.v1; => google.events.cloud.pubsub.v1
-    sed -i $MACOS 's/package /option java_package = \"com./' PROTO_PACKAGE
-    sed -i $MACOS 's/;/"/'  PROTO_PACKAGE
-    echo -en "\n$(cat PROTO_PACKAGE);" >> $proto_src
-
-    echo "# - proto bindings"
-    $PROTOC_PATH \
-        -I=$PROTOBUF_SRC \
-        -I=$THIRDPARTY \
-        --java_out=$PROTOC_OUT \
-        $proto_src
-
-    echo "# - validation tests"
-    # Note: --java-snowpea_opt is a list tests to skip due to imcompatibility 
-    $PROTOC_PATH \
-        --plugin=protoc-gen-java-snowpea=protoc-gen-java-snowpea/startup-script.sh \
-        --java-snowpea_out $TEST_GEN_OUT \
-        --java-snowpea_opt "MessagePublishedData,LogEntryData#strict" \
-        -I $PROTOBUF_SRC \
-        -I $THIRDPARTY \
-        --experimental_allow_proto3_optional \
-        $proto_src
-}
-
-for i in $(find "${PROTOBUF_SRC}" -type f -name data.proto); do
-  _generate $i
+echo
+echo "########################################################"
+for proto_src in $(find "${PROTOBUF_SRC}" -type f -name data.proto); do
+  echo
+  echo "# Generating proto bindings - $proto_src"
+
+  # Comment out preset Java options
+  sed -i $MACOS 's/option java/\/\/option java/' $proto_src
```

Comment:
I know this is original, but recommend double quotes to prevent globbing and word splitting (are we linting our bash with shell script at all?) 

so 

sed -i "$MACOS" 's/option java/\/\/option java/' "$proto_src"

Reply:
Captured request: https://github.com/googleapis/google-cloudevents-java/issues/133